packages/office-ui-fabric-react/src/components/__snapshots__/Layer.Basic.Example.tsx.shot

```diff
@@ -194,7 +200,7 @@ exports[`Component Examples renders Layer.Basic.Example.tsx correctly 2`] = `
       Hello world.
     </div>
     <div>
-      04:41:20
+      9:41:20 PM
```

Comment:
@JasonGore I've been having date formatting issues with snapshots. Any idea why? For this one and DatePicker.Bounded.Example.

Reply:
Hmm, it looks like `Date().toLocaleTimeString()` is varying in output based on platform... I'm on it now and will get a fix in ASAP

Reply:
Thank you!